### PR TITLE
15839 - E2e tests (mock data) for all of Org/Receiver/Sender Settings Pages

### LINKED ISSUE #15839: E2e tests (mock data) for all of Org/Receiver/Sender Settings Pages
## Problem statement
E2E tests (mock data) for Org Settings page, including the Receiver Edit and Sender Edit pages and their functionality.

## What you need to know
For example: https://staging.reportstream.cdc.gov/admin/orgsettings/org/ignore
Feel free to use the Ignore org on staging

## Acceptance criteria
For Org Settings:
- [x] Test for not authenticated scenario
- [x] authenticated receiver returns Page Not Found
- [x] authenticated sender returns Page Not Found
- [x] authenticated admin when there is an error and error displays
- [x] authenticated admin when there is no error has correct title
- [x] authenticated admin when there is no error edit section is populated with Meta, Description, Jurisdiction, and Filters.
- [x] Test for Footer
- [x] For IGNORE org, Meta, Description, and Jurisdiction all populate with some type of non-error data
- [x] For IGNORE org, Organization Sender Settings table loads without error and has at least 1 entry populated
- [x] For IGNORE org, Organization Receiver Settings table loads without error has at least 1 entry populated
- [x] All buttons function as expected without error

For a Sender Edit page:
- [x] Page loads without errors
- [x] Org name populates
- [x] Sender name populates
- [x] Tests Button functionality (except for Delete)

For a Receiver Edit page:
- [x] Page loads without errors
- [x] Org name populates
- [x] Sender name populates
- [x] Tests Button functionality (except for Delete)

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

-E2e tests (mock data) for all of Org/Receiver/Sender Settings
- Fixed last Mile Failures e2e broken test

Test Steps:
1. Run e2e

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #15839

### CODE REVIEW

File: frontend-react/e2e/spec/all/authenticated/admin/organization-settings-page.spec.ts [hunk 1]
```diff
@@ -48,60 +48,60 @@ test.describe("Admin Organization Settings Page", () => {
...
+        test("if there is an error, the error is shown on the page", async ({organizationPage}) => {
             organizationPage.mockError = true;
             await organizationPage.reload();
             await expect(organizationPage.page.getByText("there was an error")).toBeVisible();
         });
 
-        test.describe("When there is no error", () => {
+        test.describe("when there is no error", () => {
             test("nav contains the 'Admin tools' dropdown with 'Organization Settings' option", async ({
-                                                                                                           organizationPage,
```
Comment (jpandersen87) [inline frontend-react/e2e/spec/all/authenticated/admin/organization-settings-page.spec.ts:82]: the indentation got lost here

**Review by jpandersen87 — APPROVED** [review]

File: frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+        test.describe("when there is an error", () => {
+            test("the error is shown on the page", async ({ organizationEditPage }) => {
+                organizationEditPage.mockError = true;
+                await organizationEditPage.reload();
+                await expect(organizationEditPage.page.getByText("there was an error")).toBeVisible();
+            });
+        });
+
+        test.describe("when there is no error", () => {
+            test("has correct title", async ({ organizationEditPage }) => {
```
[the remark this answers is not in the document]
Comment (etanb) [reply frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts:79]: https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/e2e/spec/all/public/about/about-page.spec.ts#L40-L44

[the remark this answers is not in the document]
Comment (etanb) [reply frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts:79]: You can see an example here:

https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/e2e/spec/all/public/about/about-page.spec.ts#L40-L44

[the remark this answers is not in the document]
Comment (penny-lischer) [reply frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts:79]: No, because this page does not have a heading.

**Review by etanb — APPROVED** [review]

File: frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        test.use({ storageState: "e2e/.auth/receiver.json" });
+        test("returns Page Not Found", async ({ organizationEditPage }) => {
+            await expect(organizationEditPage.page).toHaveTitle(/Page Not Found/);
+        });
+    });
+
+    test.describe("sender user", () => {
+        test.use({ storageState: "e2e/.auth/sender.json" });
+        test("returns Page Not Found", async ({ organizationEditPage }) => {
+            await expect(organizationEditPage.page).toHaveTitle(/Page Not Found/);
```
[the remark this answers is not in the document]
Comment (penny-lischer) [reply frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts:57]: Yes, this works b/c the <title> tag contains "Page Not Found | ReportStream - CDC's free, interoperable data transfer platform"
What you have captured is the <h1> tag.

[the remark this answers is not in the document]
Comment (etanb) [reply frontend-react/e2e/spec/all/authenticated/admin/organization-settings-edit-page.spec.ts:57]: Oh that's strange -- might make sense to turn this into a string in here and reference it in the app code and the e2e code:

https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/content/error/ErrorMessages.ts

**Pull request merged**